package/docker/httpd.conf

```diff
@@ -29,7 +29,7 @@
 # least PidFile.
 #
 ServerRoot "/usr/local/apache2"
-
+X-Robots-Tag: noindex, nofollow
```

Comment:
I think we would need to add as a header attribute. https://developers.google.com/search/docs/advanced/robots/robots_meta_tag#xrobotstag-implementation